.gitattributes

```diff
@@ -0,0 +1 @@
+* -text
```

Comment:
Is that all it takes? Wow :)

Reply:
Yeah :) it tells git to interpret all files as non-text (=binary) and thus end of line normalization will not be applied